de-lib/src/main/java/org/iplantc/de/commons/client/views/dialogs/ErrorDialog.ui.xml

```diff
@@ -55,8 +55,7 @@
                                      collapsible="true">
                         <container:VerticalLayoutContainer ui:field="detailsContainer">
                             <container:child layoutData="{topData}">
-                                <gxtForm:TextArea ui:field="descriptionArea"
-                                                  readOnly="true"/>
+                                <g:HTML ui:field="descriptionArea" />
```

Comment:
this might be overwhelmingly minor, but how does this change from text area to HTML affect our users' ability to copy-paste from this area into bug reports? A lot of the time this box's contents make it in, so we should make sure it's still reasonably copyable